taskana-adapter-camunda-spring-boot-example/src/main/resources/application.properties

```diff
@@ -45,13 +45,16 @@ taskana.datasource.driverClassName = org.h2.Driver
 taskana.datasource.username = sa
 taskana.datasource.password = sa
 taskana.schemaName=TASKANA
+taskana.lockDuration=300
```

Comment:
This is the lock duration for the events, right? Should we maybe name it taskana.adapter.events.lockDuration? To be a little more consistent with e.g. taskana.adapter.scheduler.run.interval etc.

Reply:
Yes, sounds great!